cmake/check_crt_version.cmake

```diff
@@ -0,0 +1,26 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+
+function(check_crt_version)
+    include(expected_crt_version)
+
+    file(READ "${aws-crt-cpp_SOURCE_DIR}/VERSION" CRT_SIMPLE_VERSION)
```

Comment:
this variable probably won't be set if you are importing the CRT library instead of building it as a part of the SDK build, so in case of import, you want to disable the check
```
-DENABLE_CRT_VERSION_CHECK=OFF
```